### PR TITLE
revert: remove inconsistent use of outcome enums, add another error v…

### PR DESCRIPTION
**Pull request by drahnr**
…ariant instead

### CODE REVIEW

File: src/rpm/package.rs [hunk 1]
```diff
@@ -222,7 +212,7 @@ impl RPMPackage {
 
         verifier.verify(header_and_content_cursor, signature_header_and_content)?;
 
-        Ok(SignatureVerificationOutcome::Pass)
+        Ok(())
     }
 
     pub fn verify_digest(&self) -> Result<DigestVerificationOutcome, RPMError> {
```
Comment (dralley) [inline src/rpm/package.rs:218]: Need to remove it from this signature too

**Comment by dralley** [comment]
@drahnr I've merged this into https://github.com/rpm-rs/rpm/pull/124 and fixed the compile error

**Comment by drahnr** [comment]
Ok, next time, could we keep it open until the PR you merged the changeset into actually made it into `main`/`master`?

**Comment by dralley** [comment]
I don't ever use squash-and-merge personally

**Pull request closed without merging**